src/main/java/AlexLink/HomeWork/Interface/Robot/Hands/SamsungHand.java

```diff
@@ -0,0 +1,27 @@
+package AlexLink.HomeWork.Interface.Robot.Hands;
+
+public class SamsungHand implements IHand{
+
+    private int price;
+
+    public SamsungHand(int price) {
+        this.price = price;
+    }
+
+    public SamsungHand() {
```

Comment:
No default constructors for all hands, legs and heads. Price is required

Reply:
fixed